src/allmydata/storage_client.py

```diff
@@ -829,10 +842,10 @@ def get_available_space(self):
         version = self.get_version()
         if version is None:
             return None
-        protocol_v1_version = version.get(b'http://allmydata.org/tahoe/protocols/storage/v1', {})
-        available_space = protocol_v1_version.get('available-space')
+        protocol_v1_version = version.get('http://allmydata.org/tahoe/protocols/storage/v1', UnicodeKeyDict())
+        available_space = protocol_v1_version.get(u'available-space')
```

Comment:
Any particular reason for the `u` prefix on this literal (and the one below)?  It seems to break the pattern.

Reply:
Oh, I added those when that was still a non-ported module on Python 2. Can be deleted.